src/frontend/src/binder/insert.rs

```diff
@@ -113,10 +116,55 @@ impl Binder {
             }
         };
 
+        let mut target_table_col_idxs: Vec<TableIdx> = vec![];
+        for query_column in &columns {
+            let column_name = &query_column.value; // value or real_value() ?
```

Comment:
I am not sure what to use: `value` or `real_value`. Could you help me out?

Reply:
Considering this case, we need to support case insensitive here. 
```sql 
create table t (a int, b int);
insert into t (A, B) values (1,2);
```

Reply:
I think the easiest way to support case insensitivity is to just cast it to lower case, right? Other than that `real_value` is also related to case insensitivity. The docs of the function read: 

> Value after considering quote style In certain places, double quotes can force case-sensitive, but not always e.g. session variables.

Reply:
Using `real_value()` without casting to lower case also works for my test case. What do you prefer @chenzl25?